src/main/java/sc/fiji/bdvpg/bdv/navigate/AdaptedViewCenterTransformCreator.java

```diff
@@ -0,0 +1,65 @@
+package sc.fiji.bdvpg.bdv.navigate;
+
+import bdv.util.BdvHandle;
+import net.imglib2.realtransform.AffineTransform3D;
+
+public class AdaptedViewCenterTransformCreator
+{
+	private final BdvHandle bdvHandle;
+	private final double[] xyz;
+	private final int t;
+
+	/**
+	 * Based on the current ViewerTransform, creates a new viewer transform with a new center position.
```

Comment:
Maybe move this to the header and put the documentation like the other actions.

Also I think we were trying to implement Runnable.

Reply:
Docu: OK
Runnable: I don't think this should be runnable (see my answer below).

Reply:
...but we could make it `Callable` if that helps, returning an `AffineTransform`.